pubspec.yaml

```diff
@@ -35,11 +35,16 @@ dependencies:
   youtube_player_flutter: ^7.0.0+7
   screen: ^0.0.5 # keeping screen on
   get_it: ^5.0.6
+  provider: ^4.3.2
+  hardware_buttons: ^1.0.0
+  google_sign_in:
```

Comment:
Why we have not added **google_sign_in** package's version here? 

https://pub.dev/packages/google_sign_in/install

Reply:
If I specify the current version of google_sign_in, there raise a compatibility error..... So I don't write it..... Without version name, it takes the compatible version....so no problem will raise here